tools/splat.yaml

```diff
@@ -4878,11 +4884,11 @@ segments:
       - [0xC047F0, "bin"]
       - [0xC20C70, "bin"] # rodata
       - [0xC20CF8, "bin"] # rodata
-      - [0xC20F40, "bin"] # rodata
+  - [0xC20F40, "bin"]
   - name: world/area_dgb/dgb_02/
     type: code
     overlay: True
-    start: 0xC2EBA0
+    start: 0xC2EBA0 # this is right
```

Comment:
If this is right, do we need a comment saying so? :p

Reply:
lol oops